docs/product-administration-and-configuration-guide/src/main/asciidoc/chap-business-central-configuration.adoc

```diff
@@ -195,6 +195,11 @@ $ mv web-exec-server.xml web.xml
 * `Dorg.kie.active.profile=ui-server` - To activate UI server profile
 endif::BPMS[]
 
+[[_sect_configuring_standalone_business_central]]
+== Configuring Standalone Business Central
+
+If you run Business Central in standalone mode, you must configure a connection to an execution server so that the screens and perspectives related to monitoring can display data. If Business Central is not connected to an execution server, it will not have access to data.
```

Comment:
Even though the feature jira talks about "Ability for users to use Stand-alone business central with no attached execution server" there is not such thing as standalone mode (which you could use as opposed to some other mode). The story behind this is that in version 7 the runtime environment for rules and processes which used to be embedded in business-central has moved to separate war file (kie-server), so it's no longer embedded in business-central.war. So from version 7 on, business-central will always be "in standalone mode" but we don't call it that way anywhere. With that in mind I'd rewrite the first sentence like so:

In order to see jBPM Process / Task data in runtime perspectives of business-central (e.g. Process Definitions, Process Instances, Tasks ...) you must configure connection to execution server. etc.